java/client/src/test/java/glide/connectors/resources/ThreadPoolResourceAllocatorTest.java

```diff
@@ -0,0 +1,43 @@
+package glide.connectors.resources;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.mockito.internal.verification.VerificationModeFactory.times;
+
+import io.netty.channel.EventLoopGroup;
+import java.util.function.Supplier;
+import org.junit.jupiter.api.Test;
+
+public class ThreadPoolResourceAllocatorTest {
+
+    ThreadPoolResourceAllocator service;
+
+    @Test
+    public void getOrCreateReturnsDefault() {
+        new ThreadPoolResourceAllocator.ShutdownHook().run();
```

Comment:
what is this for?

Reply:
I will remove it as we will make the access modifier private.